scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -172,10 +190,10 @@ If it does, then we have pretty good evidence that there _is_ a cell in there.
 
 This test was first proposed by [Lun _et al._ (2019)](https://doi.org/10.1186/s13059-019-1662-y) and implemented as `emptyDrops()` in the `DropletUtils` package. 
 This method was then adopted, with some modifications, as the default cell filtering method used by Cell Ranger.
-Here we will use the `emptyDropsCellRanger()` function to perform filtering that most closely matches the Cell Ranger implementation.
+Here we will use the [`emptyDropsCellRanger()` function](https://rdrr.io/github/MarioniLab/DropletUtils/man/emptyDropsCellRanger.html) to perform filtering that more closely matches the Cell Ranger implementation.
```

Comment:
I was wondering about whether the `bioc` rdrr link should be used ( https://rdrr.io/bioc/DropletUtils/) instead but it looks like that site is very out of date (several minor versions behind) and doesn't include this function!
I'll note though that this old version has the best maintainer contact info of all versions. It's not close.